.idea/httpRequests/http-requests-log.http

```diff
@@ -0,0 +1,197 @@
+POST https://anfisa.forome.dev/lihander-dev/redeploy
```

Comment:
Don't create one pull request for a several tasks, your PR consists of 25k strings changes, it's not possible to check it.

Reply:
Please create separate pull requests for each task